src/Raven.Client/Documents/Linq/RavenQueryProviderProcessor.cs

```diff
@@ -3211,6 +3229,13 @@ private string TranslateSelectBodyToJs(Expression expression)
                 }
             }
 
+            if (expression is MemberExpression memberExpression)
+            {
+                var name = GetSelectPath(memberExpression.Member);
```

Comment:
??

Reply:
I have double-checked it. The code was unnecessary, so I removed it.